From 07c72d2a2742a1289987b022c64a07d452982b33 Mon Sep 17 00:00:00 2001
From: Andrew Bayer <andrew@cloudera.com>
Date: Thu, 10 May 2012 12:55:38 -0700
Subject: [PATCH 095/795] MR1: CDH-5555. Add -lcrypto to pipes examples compilation.
 (cherry picked from commit 2cbdc1ce281a59c73e2d5e66706a7ffe6dfe8f2d)

(cherry picked from commit 9c41416ec52dea5570d18fce2cd53cbccd66cc57)
---
 src/examples/pipes/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/examples/pipes/Makefile.am b/src/examples/pipes/Makefile.am
index 731ab1e..d2626d0 100644
--- a/src/examples/pipes/Makefile.am
+++ b/src/examples/pipes/Makefile.am
@@ -17,7 +17,7 @@ ACLOCAL_AMFLAGS = -I ../../c++/utils/m4
 AM_CXXFLAGS=-Wall -I$(HADOOP_UTILS_PREFIX)/include \
             -I$(HADOOP_PIPES_PREFIX)/include
 LDADD=-L$(HADOOP_UTILS_PREFIX)/lib -L$(HADOOP_PIPES_PREFIX)/lib \
-      -lhadooppipes -lhadooputils
+      -lhadooppipes -lhadooputils -lcrypto
 
 bin_PROGRAMS= wordcount-simple wordcount-part wordcount-nopipe pipes-sort
 
-- 
1.7.0.4

